src/BinaryParsers/PEBinary/PEBinary.cs

```diff
@@ -6,12 +6,13 @@
 using System.Collections.Generic;
 using System.IO;
 
+using Microsoft.CodeAnalysis.BinaryParsers.Dwarf;
 using Microsoft.CodeAnalysis.BinaryParsers.PortableExecutable;
 using Microsoft.CodeAnalysis.BinaryParsers.ProgramDatabase;
 
 namespace Microsoft.CodeAnalysis.BinaryParsers
 {
-    public class PEBinary : BinaryBase
+    public class PEBinary : BinaryBase, IDwarfBinary

```

Comment:
>IDwarfBinary [](http://example.com/codeflow?start=40&length=12)

PE can also have Dwarf info. 
this adds some complexity to the code.

we had some concern with it and work on other priority items,
this draft PR I have not yet wok on the tests, will work on them when continue.